Src/PerformanceCollector/Perf.Shared/QuickPulseTelemetryProcessor.cs

```diff
@@ -194,7 +197,7 @@ private ITelemetryDocument ConvertRequestToTelemetryDocument(RequestTelemetry re
             {
                 try
                 {
-                    // some of the requestTelemetry properties might be deffered by using RequestTrackingTelemetryModule.DisableTrackingProperties.
+                    // some of the requestTelemetry properties might be deferred by using RequestTrackingTelemetryModule.DisableTrackingProperties.
```

Comment:
also why is this appearing inside #if net45 only?

Reply:
I believe this change was only intended for ASP.Net applications, not CORE.
As you can see I'm not changing that in my PR, that already exists.